GATracker.swift

```diff
@@ -58,22 +55,22 @@ class GATracker {
         #endif
         
         self.tid = tid
-        self.appName = NSBundle.mainBundle().infoDictionary!["CFBundleName"] as! String
-        let nsObject: AnyObject? = NSBundle.mainBundle().infoDictionary!["CFBundleShortVersionString"]
+        self.appName = Bundle.main.infoDictionary!["CFBundleName"] as! String
+        let nsObject: AnyObject? = Bundle.main.infoDictionary!["CFBundleShortVersionString"] as AnyObject?
```

Comment:
Same thing here.
`let nsObject: AnyObject? = Bundle.main.infoDictionary["CFBundleShorVersionString"] as? AnyObject` should do the job.